modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/macros/TestrayManage.macro

```diff
@@ -58,10 +66,54 @@ definition {
 		Click(locator1 = "TestrayManage#SAVE_BUTTON");
 	}
 
+	macro delete {
```

Comment:
@joelrocha68 can this macro be more assertive on what it is deleting? This delete button can be applied to anything?